lcd_test/helpers.go

```diff
@@ -357,22 +357,34 @@ func startTM(
 }
 
 // startLCD starts the LCD.
-func startLCD(logger log.Logger, listenAddr string, cdc *codec.Codec) (net.Listener, error) {
-	rs := lcd.NewRestServer(cdc)
+func startLCD(listenAddr string, cdc *codec.Codec) (*api.Server, error) {
```

Comment:
@alexanderbez can you help me to start the Rest server in the new way? Now looks like it can talk with the Rest server, but I think it does not reach the tendermint node.

The Listener in the config is the address to the LCD no? Where do we set the address of the Tendermint node?

Reply:
Debbugging is ongoing

Reply:
Solved!